### DISCUSSION TITLE
Timed videos

### DISCUSSION THREAD

**Discussion by bergagi1** [bonsai-rx/bonsai#1131] [Q&A]
Hello. I was able set a workflow which records videos only when an animal is in proximity of the IR sensor. In addition I would like to built in a timer, so that is the animal stay there for more than 1 minute ,the recording is cancelled (the mouse might fall asleep in the area close to the sensor).
the workflow is like that:

![image](https://user-images.githubusercontent.com/87769998/201329884-9ffb6f36-3039-42de-aaf2-3b6c25e87480.png)

I tried different "time" operator after takeUntil but did not manage to get what I want. Any suggestion?

**Comment by sannatitus** [comment]
If it's red, it's not connected properly to the rest of the workflow. I would reccomend using just the reactive timer node (not a grouped workflow), and connect it _before_ the TakeUntil
